mteb/tasks/retrieval/multilingual/vidore3_bench_retrieval.py

```diff
@@ -0,0 +1,592 @@
+        license="cc-by-4.0",
+        annotations_creators="derived",
+        dialect=[],
+        modalities=["text", "image"],
+        sample_creation="found",
+        bibtex_citation=r"""
+@article{mace2025vidorev2,
+  author = {Macé, Quentin and Loison António and Faysse, Manuel},
+  journal = {arXiv preprint arXiv:2505.17166},
+  title = {ViDoRe Benchmark V2: Raising the Bar for Visual Retrieval},
+  year = {2025},
+}""
+""",
+        prompt={"query": "Find a screenshot that is relevant to the user's question."},
+        is_public=True,
+    )
+
+    def load_data(self, **kwargs):
+        if self.data_loaded:
+            return
+
+        self.corpus, self.queries, self.relevant_docs = _load_data(
+            path=self.metadata.dataset["path"],
+            splits=self.metadata.eval_splits,
+            langs=["english", "french", "spanish", "german", "italian", "portuguese"],
+            # revision=self.metadata.dataset["revision"],
+        )
+
+        self.data_loaded = True
+
+
+class Vidore3EnergyNuclearRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="Vidore3EnergyNuclearRetrieval",
+        description="Retrieve associated pages according to questions.",
+        reference="https://arxiv.org/pdf/2407.01449",
+        dataset={
+            "path": "mysecretorga/energy_nuclear_plant_en",
+            "revision": "test",
+        },
+        type="DocumentUnderstanding",
+        category="t2i",
+        eval_splits=["test"],
+        eval_langs=_LANGS,
+        main_score="ndcg_at_10",
+        date=("2025-10-01", "2025-11-01"),
+        domains=["Academic"],
+        task_subtypes=["Image Text Retrieval"],
+        license="cc-by-4.0",
+        annotations_creators="derived",
+        dialect=[],
+        modalities=["text", "image"],
+        sample_creation="found",
+        bibtex_citation=r"""
+@article{mace2025vidorev2,
+  author = {Macé, Quentin and Loison António and Faysse, Manuel},
+  journal = {arXiv preprint arXiv:2505.17166},
+  title = {ViDoRe Benchmark V2: Raising the Bar for Visual Retrieval},
+  year = {2025},
+}""
+""",
+        prompt={"query": "Find a screenshot that is relevant to the user's question."},
+        is_public=False,
+    )
+
+    def load_data(self, **kwargs):
+        if self.data_loaded:
+            return
+
+        self.corpus, self.queries, self.relevant_docs = _load_data(
+            path=self.metadata.dataset["path"],
+            splits=self.metadata.eval_splits,
+            langs=["english", "french", "spanish", "german", "italian", "portuguese"],
+            # revision=self.metadata.dataset["revision"],
+        )
+
+        self.data_loaded = True
+
+
+class Vidore3TelecomTechnicalRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="Vidore3TelecomTechnicalRetrieval",
+        description="Retrieve associated pages according to questions.",
+        reference="https://arxiv.org/pdf/2407.01449",
```

Comment:
I couldn't find any examples from this dataset in the paper. It would be great to have ~5 examples public so that user can inspect the data